h2o-py/h2o/h2o.py

```diff
@@ -1257,6 +1259,102 @@ def make_metrics(predicted, actual, domain=None, distribution=None):
               data={"domain": domain, "distribution": distribution})
     return res["model_metrics"]
 
+gen_model_file_name = "h2o-genmodel.jar"
+h2o_predictor_class = "water.util.H2OPredictor"
+
+def find_file(name, path):
+    """
+    Helper function for predict_json() function to check if both MOJO and h2o-genmodel.jar are available
+    :param name: file name to find
+    :param path: folder/directory name with full path
+    :return: Successful path name or None if failed
+    """
+    for root, dirs, files in os.walk(path):
+        if name in files:
+            return os.path.join(root, name)
+
+def check_java_process_and_version():
+    """
+    Helper function for predict_json() to check java version in the machine
+    :return: True/False if support java is available
+    """
+    result = subprocess.call(["java", "-version"])
+    if result is not 0:
+        raise RuntimeError("Java process not found, exiting!!")
+
+    output = subprocess.check_output(["java", "-version"], stderr=subprocess.STDOUT)
+    #output = subprocess.Popen(["java", "-version"], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    #print output.communicate()
+    #print output.wait()
+    result = "0.0"
+    supported = False
+    if output.find("1.6.") is not -1:
+        result = "1.6"
+    elif output.find("1.7.") is not -1:
+        result = "1.7"
+        supported = True
+    elif output.find("1.8.") is not -1:
+        result = "1.8"
+        supported = True
+    return supported
+
+def predict_json(dir_path_to_genmodel_and_mojo, mojo_model_name, json_data, show_debug=False):
+    """
+    MOJO scoring function to take a pandas data frame as json string and use MOJO model as zip file to score
```

Comment:
Doesn't seem to do anything with Pandas.

Reply:
This method can take many types, no?

Reply:
It takes json input only as one row at a time. If there are more then rows then user has to feed each row in the loop.